packages/preview-middleware/templates/test/qunit.html

```diff
@@ -12,10 +12,12 @@
 		}'
 		data-sap-ui-animation="false"
 		data-sap-ui-compatVersion="edge"
-		data-sap-ui-async="false"
-		data-sap-ui-oninit="module:<%- initPath %>">
+		data-sap-ui-async="true">
 	</script>
 
+	<script src="<%- basePath %>/resources/sap/ui/thirdparty/qunit-2.js"></script>
```

Comment:
I'm afraid this setup results in a race condition between the script initializing global `Qunit` and the `data-sap-ui-oninit` module accessing it (`QUnit.config.autostart = false;`)

Reply:
It's gone again with https://github.com/SAP/open-ux-tools/commit/596ba41e7304637d5f1cd458f87437426f4d6d94